StortSpelProjekt/Engine/src/Renderer/Animation.h

```diff
@@ -12,8 +12,11 @@ struct BoneInfo
 struct TransformKey
 {
 	DirectX::XMFLOAT3 position = {};
+	DirectX::XMFLOAT3* pPosition = nullptr;
 	DirectX::XMFLOAT4 rotationQuaternion = {};
-	DirectX::XMFLOAT3 scaling = {};
+	DirectX::XMFLOAT4* pRotation = nullptr;
+	DirectX::XMFLOAT3 scaling = { 1.0f,1.0f,1.0f };
+	DirectX::XMFLOAT3* pScale = nullptr;
```

Comment:
code standard with variable names

Reply:
We have not discussed code standard for variables in structs. To me it looks okay